tests/vmi_networking_test.go

```diff
@@ -751,6 +751,7 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				Expect(err).ToNot(HaveOccurred())
 				virtHandlerIPs = virtHandlerPod.Status.PodIPs
 
+				time.Sleep(5 * time.Second)
```

Comment:
I'm aware this is unacceptable. I'll come up w/ something less wrong than this. Do we something similar on any networking tests ?